src/domain/community/community/community.service.ts

```diff
@@ -389,10 +389,14 @@ export class CommunityService {
       email
     );
 
-    return await this.communicationService.getCommunityRoom(
+    const room = await this.communicationService.getCommunityRoom(
```

Comment:
Extract this in a method - it is repeated twice, exactly the same code with one parameter difference

Reply:
The next few lines... Not visible from the comment - you can see in the code.